### PR TITLE
Added rtas_dbg tests

### PR DESCRIPTION
**Pull request by shirishaganta1**
Tests the rtas_dbg command with a list of command line arguments

Signed-off-by: Shirisha Ganta <SHIRISHA.Ganta1@ibm.com>

### CODE REVIEW

**Comment by shirishaganta1** [comment]
# avocado run rtas_dbg.py 
JOB ID        : a3d4f6dac4964727241d69e6bb74ef285b32f1bc
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2021-08-03T16.34-a3d4f6d/job.log
 (1/2) rtas_dbg.py:rtas_dbg.test_rtas_dbg: PASS (0.77 s)
 (2/2) rtas_dbg.py:rtas_dbg.test_negative_rtas_dbg: PASS (0.86 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2021-08-03T16.34-a3d4f6d/results.html
JOB TIME    : 2.13 s

File: ras/rtas_dbg.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    @skipIf(IS_POWER_NV, "This test is supported on PowerVM environment")
+    def setUp(self):
+        sm = SoftwareManager()
+        if not sm.check_installed("powerpc-utils-core") and \
+           not sm.install("powerpc-utils-core"):
+            self.cancel("Fail to install required 'powerpc-utils-core' package")
+
+    def test_rtas_dbg(self):
+        list = ['-l', '-l 14', '-l get-power-level']
```
[the remark this answers is not in the document]
Comment (shirishaganta1) [reply ras/rtas_dbg.py]: okay will add yaml file

File: ras/rtas_dbg.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        for list_item in list:
+            cmd = "rtas_dbg %s" % list_item
+            if process.system(cmd, ignore_status=True, sudo=True):
+                self.log.info("%s command failed" % cmd)
+                self.fail("rtas_dbg: %s command failed to execute" % cmd)
+
+    def test_negative_rtas_dbg(self):
+        # Negative tests
+        list = ['-l get-power', '-l 32']
```
[the remark this answers is not in the document]
Comment (shirishaganta1) [reply ras/rtas_dbg.py]: okay

**Comment by shirishaganta1** [comment]
yaml file
# avocado run rtas_dbg.py -m rtas_dbg.py.data/options.yaml
JOB ID     : 8d8e63e06b02cd9dbf1e41307da3fbda4994ece2
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2021-08-04T11.00-8d8e63e/job.log
 (1/2) rtas_dbg.py:rtas_dbg.test_rtas_dbg;run-ec6e: PASS (0.74 s)
 (2/2) rtas_dbg.py:rtas_dbg.test_negative_rtas_dbg;run-ec6e: PASS (0.90 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2021-08-04T11.00-8d8e63e/results.html
JOB TIME   : 2.20 s

**Comment by shirishaganta1** [comment]
sles
\# avocado run rtas_dbg.py -m rtas_dbg.py.data/options.yaml 
JOB ID     : cc3972570b5be60fdeeceefc7b82e2cce20ec149
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2021-08-05T11.05-cc39725/job.log
 (1/2) rtas_dbg.py:rtas_dbg.test_rtas_dbg;run-ec6e: PASS (0.26 s)
 (2/2) rtas_dbg.py:rtas_dbg.test_negative_rtas_dbg;run-ec6e: PASS (0.25 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2021-08-05T11.05-cc39725/results.html
JOB TIME   : 0.80 s

Rhel
\# avocado run rtas_dbg.py -m rtas_dbg.py.data/options.yaml
JOB ID     : afdebf30be35cba38ddcf8985af320f1bf590c2a
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2021-08-05T11.06-afdebf3/job.log
 (1/2) rtas_dbg.py:rtas_dbg.test_rtas_dbg;run-ec6e: PASS (0.71 s)
 (2/2) rtas_dbg.py:rtas_dbg.test_negative_rtas_dbg;run-ec6e: PASS (0.75 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2021-08-05T11.06-afdebf3/results.html
JOB TIME   : 1.89 s

**Review by PraveenPenguin — APPROVED** [review]

**Pull request merged**